packages/components-native/src/Chip/Chip.style.ts

```diff
@@ -29,6 +29,6 @@ export const styles = StyleSheet.create({
     padding: tokens["space-smaller"],
   },
   activeDismissIcon: {
-    backgroundColor: tokens["color-surface--background"],
+    backgroundColor: tokens["color-interactive--background"],
```

Comment:
I don't actually recall _why_ we need a different background color here, but that's probably a question for another day